systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeAbstractST.java

```diff
@@ -34,43 +22,19 @@
 @ExtendWith(VertxExtension.class)
 @Tag(BRIDGE)
 @Tag(REGRESSION)
-@Tag(NODEPORT_SUPPORTED)
-@Tag(EXTERNAL_CLIENTS_USED)
 public class HttpBridgeAbstractST extends AbstractST {
-    private static final Logger LOGGER = LogManager.getLogger(HttpBridgeAbstractST.class);
-
-    protected WebClient client;
-    protected String bridgeExternalService = CLUSTER_NAME + "-bridge-external-service";
+    public static int bridgePort = Constants.HTTP_BRIDGE_DEFAULT_PORT;
+    public static String bridgeHost = "";
+    public static String kafkaClientsPodName = "";
+    public static String bridgeServiceName = KafkaBridgeResources.serviceName(CLUSTER_NAME);
```

Comment:
Wouldn't be better to use `KafkaBridgeResources.url()` instead just service name?

Reply:
For this case I'm gonna add `bridgeUrl` var to use `KafkaBridgeResources.url()` method and I will keep `bridgeServiceName` var for example clients usage.